packages/integrations/image/README.md

```diff
@@ -195,6 +220,8 @@ A `number` can also be provided, useful when the aspect ratio is calculated at b
 **Default:** `undefined`
 </p>
 
+> This is not supported by the default Squoosh service. See the [installation section](#installing-sharp-optional) for details on using the `sharp` service instead.
```

Comment:
```suggestion
:::note
This is not supported by the default Squoosh service. See the [installation section](#installing-sharp-optional) for details on using the `sharp` service instead.
:::
```
@delucis doesn't let me abuse `blockquote` anymore... 😅 

This is the format for asides that we use in Docs. If you want a custom title (instead of just `note`), then you can use the format 

```
:::note[custom title that will be in all caps]
text
:::
```

Whatever you do here, should be done in all six instances of this (3 properties of `<Image />`, 3 properties of `<Picture />`)

Reply:
How does `:::note` work in NPM?  I'm wondering if we eventually want a README that goes to NPM and a separate DOCS.md that is hosted on the docs site 🤔

Reply:
We eventually *do* want most of this info to live in Docs... 😅  Let me see if I can find an example of one of our Asides that is currently in a README, so we can check to be sure what it looks like...

Reply:
The very top of this file uses: 

```
> ⚠️ This integration is still experimental! Only node environments are supported currently, stay tuned for Deno support in the future!
```

So, I guess there's precedent for that for now. We don't need to hold this up over the Aside/Blockquotes. And, I just discovered that GitHub's own Note syntax doesn't even render in Docs!  😱  So.... let's not do that!